envoy/server/overload/overload_manager.h

```diff
@@ -52,6 +52,19 @@ class OverloadActionStatsNameValues {
 
 using OverloadActionStatsNames = ConstSingleton<OverloadActionStatsNameValues>;
 
+/**
+ * A point that provides context on whether to shed load.
```

Comment:
What does "point" mean here? It seems like this class exists to determine if load should be shed and that's about it. So maybe this is a Decider or Chooser, or Policy or Selector? LoadShedPolicy? Not sure. What do you think?

Reply:
I was going for something similar to the terminology of https://docs.kernel.org/trace/tracepoints.html

I've added more details to the comment to provide more context. I think the concrete implementations will further make this less  abstract

Effectively the usage could look like:

```
if(overload_manager->getLoadShedPoint('foo_stage_of_request')->shouldShedLoad()) {
    abort;
}

// Allow processing
```